README.md

```diff
@@ -30,6 +30,16 @@ inside a `git` repository.
 To specify custom application roots (such as with the `src` pattern) pass a
 colon-separated `--application-directories` parameter.
 
+```console
+$ seed-isort-config  --application-directories=src:.
+```
+
+Specific files may be excluded from the process. For instance:
+
+```console
+$ seed-isort-config --exclude='tests/.*\.py'
+```
```

Comment:
I think this is adequately covered below with the pre-commit example :man_shrugging:

Reply:
My experience is that people don't read all the docs or the whole Read Me, only parts of it. Having the example in the command line and the hook feels worthwhile to me, but if you feel strongly I can remove this section

Reply:
Might even be able to just replace this whole section with the output of `seed-isort-config --help`

Reply:
I looked at `--help` while trying to get this working, and still got the use of `application-directories` wrong (I initially tried `--application-directories src/verified_user:test`, which does not work). It wasn't until you gave me an example in the issue that it started working on my end.

Reply:
heh, well the default of `.` should have been a hint -- is that in the help text?